app/lib/features/chat_ng/dialogs/message_actions.dart

```diff
@@ -22,29 +28,36 @@ void reactionSelectionAction({
     barrierColor: Colors.transparent,
     transitionDuration: const Duration(milliseconds: 200),
     pageBuilder: (context, animation, secondaryAnimation) {
-      return Stack(
+      return Column(
```

Comment:
Moved from ```Stack``` to ```Column``` layout. Because it allows better positioning for message actions + reaction selector and makes the selected message appeared centered regardless of its actual position in message list.